docs/stories/customComponents/List.stories.js

```diff
@@ -86,7 +87,7 @@ function ListStory() {
   const subtitle = text('Subtitle', 'The most successful French Top Chefs');
   const buttonProps = {
     color: 'secondary',
-    icon: true,
+    icon: <Plus height="11px" width="11px" fill="#007EFF" />,
```

Comment:
You have some `prop-type` errors.
![Screenshot 2020-01-24 at 11 12 25](https://user-images.githubusercontent.com/13311463/73061446-7cecc980-3e9a-11ea-8a64-a386ba1eaf5f.png)